functions/go/create-setters/createsetters/create_setters_test.go

```diff
@@ -0,0 +1,235 @@
+package createsetters
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/kustomize/kyaml/kio"
+	kyaml "sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+func TestCreateSettersFilter(t *testing.T) {
```

Comment:
Add more tests such as empty setters etc. Refer to apply-setters test cases and add corresponding tests here.

Reply:
Sure